packages/@aws-cdk/aws-s3-deployment/README.md

```diff
@@ -30,7 +30,7 @@ const websiteBucket = new s3.Bucket(this, 'WebsiteBucket', {
 });
 
 new s3deploy.BucketDeployment(this, 'DeployWebsite', {
-  source: s3deploy.Source.asset('./website-dist'),
+  sources: [s3deploy.Source.asset('./website-dist')],
```

Comment:
Add a little description on what happens when multiple sources are defined (each one is uploaded individually and merged during deployment)